python-packages/order_utils/src/zero_ex/json_schemas/schemas/zero_ex_transaction_schema.json

```diff
@@ -0,0 +1,10 @@
+{
+    "id": "/zeroExTransactionSchema",
+    "properties": {
+        "data": { "$ref": "/hexSchema" },
+        "signerAddress": { "$ref": "/addressSchema" },
+        "salt": { "$ref": "/wholeNumberSchema" }
+    },
+    "required": ["data", "salt", "signerAddress"],
+    "type": "object"
+}
```

Comment:
@feuGeneA did you simply copy over all schemas?

Reply:
Yup.

Reply:
Note that the diff command in the linter run will ensure that there are no differences between the copies.